File: Iniciante/1035 - Teste de Seleção 1/C#/1035 - Teste de Seleção 1.cs
using System;

class URI {
    static void Main(){
        string[] teste = Console.ReadLine().Split(' ');
            int A = Int32.Parse(teste[0]);
            int B = Int32.Parse(teste[1]);
            int C = Int32.Parse(teste[2]);
            int D = Int32.Parse(teste[3]);

        if((B>C) && (D>A) && (C+D > A+B) && (C>=0) && (D>=0) && (A%2 == 0)) {
            Console.WriteLine("Valores aceitos");
        } else {
            Console.WriteLine("Valores nao aceitos");
        }
    }
}

File: Iniciante/1035 - Teste de Seleção 1/Python/1035 - Teste de Seleção 1.py
# pega 4 valores da mesma linha e atribui às variáveis
A, B, C, D = input().split(' ')

# converte os valores para o tipo inteiro
A = int(A)
B = int(B)
C = int(C)
D = int(D)

if B>C and D>A and C+D>A+B and C>0 and D>0 and A%2==0:
    print('Valores aceitos')

else:
    print('Valores nao aceitos')